Source/HYPFormsCollectionViewDataSource.m

```diff
@@ -78,11 +79,7 @@ - (instancetype)initWithCollectionView:(UICollectionView *)collectionView
 
 - (NSMutableArray *)collapsedForms
 {
-    if (_collapsedForms) return _collapsedForms;
-
-    _collapsedForms = [NSMutableArray new];
-
-    return _collapsedForms;
+    return _collapsedForms = (_collapsedForms) ?: [NSMutableArray new];
```

Comment:
@attila-at-hyper actually, it would be like
`_collapsedForms = (_collapsedForms) ? _collapsedForms : [NSMutableArray new];`